mailet/standard/src/main/java/org/apache/james/transport/mailets/RemoveMimeHeader.java

```diff
@@ -63,21 +63,24 @@ public String getMailetInfo() {
 
     @Override
     public void service(Mail mail) throws MessagingException {
-        MimeMessage  message = mail.getMessage();
-        
+        MimeMessage message = mail.getMessage();
+
+        boolean headersToRemove = message.getMatchingHeaderLines(headers.toArray(new String[headers.size()]))
```

Comment:
the usage is usually `toArray(new String[0])`, the argument help figure out the return type but has a zero size